site/content/presentations-material.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Presentations Material"
+menu = "main"
+weight = 620
++++
+
+In this page you can find some links to presentations material about Eclipse Hono showed during events, conferences and meetups.
+
+## "Eclipse Hono: Connect. Command. Control." at Virtual IoT Meetup
+
+[Virtual IoT Meetup](https://www.meetup.com/Virtual-IoT/events/242190258/) hosted by Eclipse Foundation (October 11, 2017)
+
+* Speakers : Kai Hudalla (Chief Software Architect at BoschSI), [Paolo Patierno](https://twitter.com/ppatierno) (Senior Software Engineer at Red Hat)
```

Comment:
please make it `Bosch Software Innovations` and (at least for me) leave out the *title*.